src/components/pipelines-metrics/PipelinesMetricsPage.tsx

```diff
@@ -74,6 +82,8 @@ const PipelinesMetricsPage: React.FC = () => {
             grow={{ default: 'grow' }}
           >
             <PipelinesRunsNumbersChart
+              namespace={namespace}
+              pipelineName={parentName}
```

Comment:
```suggestion
              parentName={parentName}
```
as prop name is `parentName`